django/core/management/commands/makemessages.py

```diff
@@ -168,6 +168,17 @@ def normalize_eols(raw_contents):
     return '\n'.join(lines_list)
 
 
+def normalize_locale(locale_str, stdout):
```

Comment:
Before adding the warning, I would have said that it wasn't really worth adding a whole function for this.
It could have been wrapped into the list comprehension below.

Reply:
Why are you passing `stdout` if you are using `self.stdout` below (which looks broken)?

Reply:
1. Added a new funciton just to improve the readability.
2. May be wrong its wrong variable name usage. We need `self.stdout` here to print the warning.